<launch>
  <!-- Brobot data collection launcher -->
  <node pkg="brobot" type="joy_driver.py" name="joy_driver"/>
  <node pkg="brobot" type="roboclaw_driver.py" name="roboclaw_driver"/> 
  <node pkg="brobot" type="data_collection.py" name="data_collection"/> 
  <!-- node name="image_view" pkg="image_view" type="image_view" respawn="true" output="screen">
    <remap from="image" to="/camera_array/cam0/image_raw"/>
    <param name="autosize" value="true" />
  </node -->
</launch>
